### PR TITLE
upgrades phase 1 - prepare prior versions

### PR DESCRIPTION
**Pull request by julianmrodri**
* Prepare upgrades for future validations
* Creates `force-import` task
* Adds mainnet network file for versions `2.0.0`/`2.1.0`

### CODE REVIEW

File: tasks/upgrades/utils.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    if (!deployments.implementations.components[keystr as keyof IComponents]) {
+      throw new Error(
+        `Missing deployed ${keystr} implementation for version ${version} in network ${hre.network.name}`
+      )
+    } else if (
+      !(await isValidContract(
+        hre,
+        deployments.implementations.components[keystr as keyof IComponents]
+      ))
+    ) {
```
Comment (pmckelvy1) [inline tasks/upgrades/utils.ts:52]: else

File: tasks/upgrades/utils.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    throw new Error(`TradingLib contract not found in network ${hre.network.name}`)
+  }
+
+  // Check Main implementation is defined
+  if (!deployments.implementations.main) {
+    throw new Error(
+      `Missing deployed Main implementation for version ${version} in network ${hre.network.name}`
+    )
+  } else if (!(await isValidContract(hre, deployments.implementations.main))) {
+    throw new Error(`TradingLib contract not found in network ${hre.network.name}`)
```
Comment (pmckelvy1) [inline tasks/upgrades/utils.ts:26]: i think this error is wrong

**Review by pmckelvy1 — CHANGES_REQUESTED** [review]

File: tasks/upgrades/utils.ts (hunk 2 above)
Comment (julianmrodri) [reply tasks/upgrades/utils.ts:26]: Yep should be Main will fix

File: tasks/upgrades/utils.ts (hunk 1 above)
Comment (julianmrodri) [reply tasks/upgrades/utils.ts:52]: Else is the expected behavior, just move forward

**Comment by pmckelvy1** [comment]
lgtm

**Pull request merged**